test/integration-test.py

```diff
@@ -302,6 +316,44 @@ def main():
             print("\nIssused certificate for domain with bad CAA records")
             die(ExitStatus.NodeFailure)
 
+        if run_node_test("challenge-not-started.com", challenge_types[0], None, "startChallenge")\
```

Comment:
This whole chunk that tests the expired-authz-purged should be split out into its own function.